public-ui/src/app/views/pm/pm.component.ts

```diff
@@ -1,15 +1,251 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ISubscription } from 'rxjs/Subscription';
+import {DataBrowserService} from '../../../publicGenerated/api/dataBrowser.service';
+import {AchillesResult} from '../../../publicGenerated/model/achillesResult';
+import {Analysis} from '../../../publicGenerated/model/analysis';
+import {DbConstantsService} from '../../utils/db-constants.service';
 
 @Component({
   selector: 'app-physical-measurements',
   templateUrl: './pm.component.html',
-  styleUrls: ['./pm.component.css']
+  styleUrls: ['../../styles/template.css', '../../styles/cards.css', './pm.component.css']
 })
-export class PhysicalMeasurementsComponent implements OnInit {
+export class PhysicalMeasurementsComponent implements OnInit, OnDestroy {
+  title = 'Browse Program Physical Measurements';
+  pageImage = '/assets/db-images/man-standing.png';
+  private subscriptions: ISubscription[] = [];
+  loadingStack: any = [];
 
-  constructor() { }
+  // Todo put constants in a class for use in other views
+  chartType = 'histogram';
+
+  // Total analyses
+  genderAnalysis: Analysis = null;
+  raceAnalysis: Analysis = null;
+  ethnicityAnalysis: Analysis = null;
+
+  // Chart titles for our gender histograms.
+  maleGenderChartTitle =  '';
+  femaleGenderChartTitle = '';
+  otherGenderChartTitle = '';
+
+  conceptGroups = [
+    { group: 'blood-pressure', groupName: 'Mean Blood Pressure', concepts: [
+      {conceptId: '903118', conceptName: 'Systolic ', analyses: null, chartType: this.chartType,
```

Comment:
There's a lot of repeated code here, I would probably make a helper or possibly a base concept object:

```
baseConcept = {
  chartType: this.chartType,
  maleCount: 0
}

{
  ...baseConcept,
  conceptId: '',
  conceptName: '',
}
```